app/interceptors/ExceptionInterceptor.java

```diff
@@ -28,23 +30,17 @@
 import play.mvc.Results;
 
 import com.google.common.base.Throwables;
-import com.google.common.collect.Sets;
 
 @Component("exceptionInterceptor")
 public class ExceptionInterceptor implements MethodInterceptor {
 
     private static Logger logger = LoggerFactory.getLogger(ExceptionInterceptor.class);
-    
-    private static Set<Class<? extends BridgeServiceException>> quietExceptions = Sets.newHashSet();
-    static {
-        quietExceptions.add(BadRequestException.class);
-        quietExceptions.add(InvalidEntityException.class);
-        quietExceptions.add(EntityAlreadyExistsException.class);
-        quietExceptions.add(EntityNotFoundException.class);
-        quietExceptions.add(UnauthorizedException.class);
-        quietExceptions.add(NotAuthenticatedException.class);
-    }
-    
+
+    private static Set<Class<? extends BridgeServiceException>> quietExceptions = ImmutableSet.of(
```

Comment:
Not related to this change.  But can we create a marker Interface QuietException?  If I write a new quiet exception, I just implement QuietException.  I don't need to update anything in ExceptionInterceptor.